base/math.jl

```diff
@@ -1318,6 +1318,7 @@ for f in (:sin, :cos, :tan, :asin, :atan, :acos,
     end
     @eval $(f)(::Missing) = missing
 end
+atan(::Missing,::Missing) = missing
```

Comment:
This line would be more consistent with the rest of the julia code base if a space were added after the comma.  (There are a handful of exceptions, the space is there much more often than it is not.)

Reply:
Thank you, garrison! I have just added support for the cases where one of the values is missing and the other is not. I believe `::Number` is better for this than `::Any` but am not sure!

```
atan(::Missing, ::Missing) = missing
atan(::Number, ::Missing) = missing
atan(::Missing, ::Number) = missing
```

Reply:
I agree that `::Number` makes the most sense, as it makes this consistent with other two-argument math functions (e.g. `*`, `/`).